### DISCUSSION TITLE
Changing the order of Size, Date

### DISCUSSION THREAD

**Discussion by sommerper** [eza-community/eza#961] [Q&A, Howto, Help] [answered]
I just found this wonderful tool and I'm trying to configure it so that Date comes before Size:
![image](https://github.com/eza-community/eza/assets/1512048/e3497ef0-c9ac-4452-b335-7196e671ff5d)
Is that possible?

Thanks!

**Comment by gierens** [comment, marked as answer]
Not at the moment, but @PThorpe92 did some work on this a couple months ago, see #148. Maybe if you ping him there he finds the time to make a PR draft.
